packages/nextjs/test/integration/test/server.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs').promises;
+const { createServer } = require('http');
+const { parse } = require('url');
+const path = require('path');
+
+const next = require('next');
+
+const { DEBUG_MODE } = require('./utils');
+const { log } = console;
+
+/**
+ * Usage: node server.js <filename>
+ *
+ * ENV Variables:
+ * DEBUG=[bool] - enable request and application logs
+ * DEBUG_DEPTH=[int] - set logging depth
+ *
+ * Arguments:
+ * <filename> - filter tests based on filename partial match
+ */
+
+const FILES_FILTER = process.argv[2];
+
+(async () => {
+  let scenarios = await fs.readdir(path.resolve(__dirname, './server'));
+
+  if (FILES_FILTER) {
+    scenarios = scenarios.filter(file => file.toLowerCase().includes(FILES_FILTER));
+  }
+
+  if (scenarios.length === 0) {
+    console.log('No tests founds.');
+    process.exit(0);
+  } else {
+    if (DEBUG_MODE) {
+      scenarios.forEach(s => console.log(`⊙ Tests Found: ${s}`));
+    }
+  }
+
+  // Silence all the unnecessary server noise. We are capturing errors manualy anyway.
+  if (!DEBUG_MODE) {
+    console.log = () => {};
+    console.error = () => {};
```

Comment:
`console` is destructured on line 9. After that, you sometimes use `console.log` (e.g. line 32), and some others `log` (e.g. line 62). Is there any reason to go with both approaches, instead of a single one?